docs/organizations/managing-security-and-risk.md

```diff
@@ -66,9 +66,9 @@ The following section explains how to integrate with each of the supported platf
 
     To install the Jira Cloud integration:
 
-    1.  Open your organization **Security and Risk** page and click the **Configure** button. This opens the configuration page.
+    1.  Open your organization **Security and Risk** page and click the **Configure** button to open the configuration page.
 
-    1.  On the right-hand side of the page, under the Integrations list, find the Jira entry and click the **Install** button. This redirects you to Atlassian's website.
+    1.  On the right-hand side of the page, under the Integrations list, find the Jira entry and click the **Install** button to open the installation modal, then confirm by clicking **Install Jira** to proceed to Atlassian's website..
```

Comment:
I find this a rather long sentence, and should be fairly easy to break it up in two.

Also, there's a spurious full stop at the end:

```suggestion
    1.  On the right-hand side of the page, under the Integrations list, find the Jira entry and click the **Install** button to open the installation modal, then confirm by clicking **Install Jira** to proceed to Atlassian's website.
```

Reply:
```suggestion
    1.  On the right-hand side of the page, under the Integrations list, find the Jira entry and click the **Install** button to open the installation modal. Confirm by clicking **Install Jira** to proceed to Atlassian's website.
```